db/migrate/20191016050734_add_home_and_current_location_to_carrier.rb

```diff
@@ -0,0 +1,12 @@
+class AddHomeAndCurrentLocationToCarrier < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :carriers, :home_location, foreign_key: { to_table: :locations }
+    add_reference :carriers, :current_location, foreign_key: { to_table: :locations }
+
+    Carrier.all.each do |carrier|
+      carrier.update(home_location_id: carrier.location_id, current_location_id: carrier.location_id)
```

Comment:
@melissa-hong  just tinny little tip for the future(not super important for this case)...always make sure that migrations are reversible, this here will fail on `bin/rails db:rollback` :)

Reply:
oh that's a good point - sorry about that. Thanks for the review!